tools/test/examples/examples.py

```diff
@@ -77,6 +77,13 @@ def main():
                              help=("build profile file"),
                              metavar="profile")
 
+    compile_cmd.add_argument("-j", "--jobs",
```

Comment:
Is there a way of ensuring that the number supplied is zero or greater?

Reply:
See https://github.com/ARMmbed/mbed-os/blob/00f461eee01880ee185f634c6203ac7b523a7004/tools/build.py#L112

Reply:
Ok, I was trying to catch the negative integer case, but whatever...